src/modules/internal_combustion_engine_control/InternalCombustionEngineControl.cpp

```diff
@@ -355,31 +355,42 @@ int InternalCombustionEngineControl::print_usage(const char *reason)
 	PRINT_MODULE_DESCRIPTION(
 		R"DESCR_STR(
 ### Description
+		
 The module controls internal combustion engine (ICE) features including:
-ignition (on/off),throttle and choke level, starter engine delay, and user request.
+ignition (on/off), throttle and choke level, starter engine delay, and user request.
 The module publishes [InternalCombustionEngineControl.msg](../msg_docs/InternalCombustionEngineControl.md).
-The architecture is as shown below.:
+		
+The architecture is as shown below:
 ![Architecture](../../assets/diagrams/ice_control_diagram.png)
 
 ### Enabling
+
 This feature is not enabled by default needs to be configured in the
 build target for your board together with the rpm capture driver:
+
+```
 CONFIG_MODULES_INTERNAL_COMBUSTION_ENGINE_CONTROL=y
 CONFIG_DRIVERS_RPM_CAPTURE=y
+```
 
 Additionally, to enable the module:
-- set [ICE_EN](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#ICE_EN)
+- Set [ICE_EN](../advanced_config/parameter_reference.md#ICE_EN)
```

Comment:
`.md` links are much better as they will link into docs for the current version. If needed we could update the parser to strip out links done this way.